actions/bulk_link_multithreading.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Could we adapt the existing `bulk_link_issue` action to support multithreading either natively or with a flag instead of creating a new action? Adding a new action that does basically the same thing seems a bit redundant. Are you open to seeing if we can refactor `bulk_link_issue`?

Reply:
Sure I can look into that. Thanks for your feedback